docs/get-started/_index.md

```diff
@@ -1,36 +1,97 @@
+  - "/product-overviews/"
+  - "/viam/"
+  - "/viam/app.viam.com/"
+imageAlt: "/general/understand.png"
+images: ["/general/understand.png"]
+carouselscript: true
 ---
 
-Learn about the Viam platform by reading our Viam in 3 minutes guide or try it out for yourself:
+Viam is a software platform that makes it easy to work with hardware and software.
 
-{{< cards >}}
-{{% card link="/get-started/viam/" %}}
-{{% card link="/get-started/try-viam/" %}}
-{{% card link="/get-started/quickstarts/control-motor/" %}}
-{{% card link="/get-started/quickstarts/collect-data/" %}}
-{{% card link="/get-started/quickstarts/detect-people/" %}}
-{{% card link="/get-started/quickstarts/write-code/" %}}
-{{% card link="/get-started/quickstarts/drive-rover/" %}}
-{{< /cards >}}
+<div>
+{{< imgproc src="/viam/viam.png" alt="Viam overview" resize="800x" class="aligncenter" >}}
+</div>
 
-If you're ready to try it on your own machine, start by installing `viam-server`.
-If you need any help along the way, join our community:
+<div>
+{{< imgproc src="/viam/pet-feeder.png" alt="Machine components" resize="400x" class="alignright" >}}
+</div>
 
-<!-- markdownlint-disable-file MD034 -->
+At the core of Viam is the open-source `viam-server` executable which runs on a device and manages hardware and software for a machine.
+To use Viam with a machine, you create a configuration that contains components and services and you can choosen from a wide variety of available models for different resources.
+For example, if you have a pet-feeding machine you might install `viam-server` on a Raspberry Pi and create a machine configuration using a camera model and a servo model for the hardware you are using.
 
-{{< cards >}}
-{{% card link="/get-started/installation/" %}}
-{{% manualcard link="https://discord.gg/viam" webm_src="/heart.webm" mp4_src="/heart.mp4" alt="A robot drawing a heart" %}}
+The Viam App provides the user interface for configuring your machine.
+It allows you to configure builtin drivers or drivers from the Viam Registry.
+`viam-server` then manages and runs the drivers for the configured resources.
+As you configure your machine, you can test each resource to confirm it is working as expected.
+
+{{<gif webm_src="/test.webm" mp4_src="/test.mp4" alt="Test a camera stream" max-width="700px" class="aligncenter">}}
+
+Viam is built to be extensible, allowing you to create your own resources for custom hardware or software and deploy your code using modules through the Viam Registry.
+
+On top of configuration, the Viam App also provides several higher-level functionalities for your machine, such as:
+
+- **Data Management**: Any data on your machine can be synced to the cloud, from where you can query it using SQL, MQL, or with code.
+- **Machine Learning**: Train machine learning models on collected data and deploy ML models to machines to enable them to intelligently see and interpret the world around them.
+- **Motion Planning**: Plan and move machine components.
```

Comment:
technically the app/cloud has nothing to do with this today, not sure if we plan on adding the ability to run motion planning in viam cloud?

Reply:
Fair enough. Can remove that line
```suggestion
```